src/components/IDO/Ido.tsx

```diff
@@ -0,0 +1,144 @@
+import AmountInput from '@components/Inputs/ExchangeAmountInput/ExchangeAmountInput'
+import { OutlinedButton } from '@components/OutlinedButton/OutlinedButton'
+import { Grid, Typography, CardMedia } from '@material-ui/core'
+import AccessTimeIcon from '@material-ui/icons/AccessTime'
+import icons from '@static/icons'
+import React, { useEffect, useState } from 'react'
+import { useStyles } from './style'
+
+export const Ido = () => {
+  const classes = useStyles()
+
+  const [est, setEst] = useState<string>(' ')
+  const [balance, setBalance] = useState<string>(' ')
+  const [inputValue, setInputValue] = useState<string | undefined>(' ')
+  const [depositedXUSD, setDepositedXUSD] = useState<string>(' ')
+
+  const [depositedUSD, setDepositedUSD] = useState<string>(' ')
+  const [depositedSOL, setDepositedSOL] = useState<string>(' ')
+  const [depositedXSOL, setDepositedXSOL] = useState<string>(' ')
+  const [depositedXBTC, setDepositedXBTC] = useState<string>(' ')
+
+  const [salePeriod, setSalePeriod] = useState<string>(' ')
+  const [gracePeriod, setGracePeriod] = useState<string>(' ')
+  const [SOLContributed, setSOLContributed] = useState<string>(' ')
+  const [estTokenPrice, setEstTokenPrice] = useState<string>(' ')
+  const [INVForSale, setINVForSale] = useState<string>(' ')
+
+  // load dummy values
```

Comment:
dummy values and handlers should be set  in storybook stories, not inside component